src/object/srv_obj_migrate.c

```diff
@@ -1570,8 +1572,8 @@ migrate_fetch_update_bulk(struct migrate_one *mrone, daos_handle_t oh,
 				fetch_eph = mrone->mo_iods_update_ephs_from_parity[i][j];
 			rc = __migrate_fetch_update_bulk(mrone, oh, &iod, 1, fetch_eph,
 						mrone->mo_iods_update_ephs_from_parity[i][j],
-						DIOF_EC_RECOV_FROM_PARITY | DIOF_FOR_MIGRATION,
-						ds_cont);
+						DIOF_EC_RECOV_FROM_PARITY | DIOF_FOR_MIGRATION |
+						DIOF_FETCH_EPOCH_EC_AGG_BOUNDARY, ds_cont);
```

Comment:
I think we'd use the fetch epoch as min(rebuild_epoch, ec_agg_boundary) when ec_agg_boundary > parity_epoch (same to what we did in migrate_fetch_update_parity()), right?

Reply:
I was just avoid other possible issue so did not change it.
But right, should be fine to change as what we discussed, let me refresh it.